### PR TITLE
Remove log level recommandation log for production

### PR DESCRIPTION
**Pull request by shouze**
Everyone can log at the level he want in fact.

### CODE REVIEW

**Comment by shouze** [comment]
ping @rdlowrey.

**Comment by kelunik** [comment]
I think we can just lower that to the level one higher than debug.

**Comment by bwoebi** [comment]
Yeah, wouldn't remove that completely and still warn if it's set to debug.

**Comment by shouze** [comment]
Makes senses I change the PR soon in this direction.

**Comment by shouze** [comment]
@kelunik @bwoebi ok done, that was exhausting 👼

File: lib/WatcherProcess.php [hunk 1]
```diff
@@ -275,7 +275,7 @@ protected function recommendLogLevel(Console $console) {
         if (!isset(Logger::LEVELS[$level])) {
             return;
         }
-        if (Logger::LEVELS[$level] > Logger::LEVELS[Logger::WARNING]) {
+        if (Logger::LEVELS[$level] > Logger::LEVELS[Logger::DEBUG]) {
             $this->logger->warning(
                 "Running aerys in production with a log level higher than \"warning\" is not " .
```
Comment (nikic) [inline lib/WatcherProcess.php]: Should probably adjust this `\"warning\"` too :)

Comment (shouze) [reply lib/WatcherProcess.php]: Ho my bad... yes of course... 👼

**Comment by shouze** [comment]
ping @nikic ok fixed 😉

**Pull request merged**